apis/core/validation/blueprint.go

```diff
@@ -62,6 +73,31 @@ func validateBlueprintImportDefinitions(fldPath *field.Path, imports []core.Impo
 			importNames.Insert(importDef.Name)
 		}
 
+		if len(importDef.Type) != 0 {
+			// type is specified, use new validation
+			switch importDef.Type {
+			case core.ImportTypeData:
+				if importDef.Schema == nil {
```

Comment:
currently we ignore other fields.
Maybe we should check if other fields are defined that do not belong to the current type.

Reply:
I added a helper function that generically checks whether all config fields which are required for the current `Type` are set and that no other config fields are set.
The method itself doesn't look that nice (using the `reflect` package), but this way extending the import/export definitions will be much nicer and easier than if we would need to add exponentially more cases to ensure that no 'wrong' config field is set.